ci/oss_compliance/osa_provided.md

```diff
@@ -0,0 +1,44 @@
+Name|Version|License
```

Comment:
is this the same list as what was used in the equivalent gloo PR, or is it specific to what each repo uses?

Reply:
It is generated per repo